functions/_pure_prompt_ssh.fish

```diff
@@ -0,0 +1,5 @@
+function _pure_prompt_ssh
+    if [ "$SSH_CONNECTION" != "" ]
```

Comment:
@schrodincat IMO the `test` builtin is more readable than`[`. I'd prefer if you use it, but whatever @edouard-lopez @rafaelrinaldi say.

Reply:
I guess @rafaelrinaldi OK with it, since he wrote it...

Reply:
It seems the idiomatic way is to use `test` command instead of brackets: https://fishshell.com/docs/current/commands.html#test. I've updated PR.

Reply:
I am happy to use `test` instead. I just wasn't aware it was a thing at the time I first wrote it.

Reply:
Prefer using `test` as it is indeed recommended by fish documentation.